test/recipes/80-test_cmp_http_data/Mock/server_withkem.cnf

```diff
@@ -0,0 +1,17 @@
+[cmp] # mock server configuration
```

Comment:
Where is this file used?

Reply:
This is a temporary test file, it was provided to pqs for testing. will be removed.